backend/zf_plugins/templates/zf_plugins/media_hero.html

```diff
@@ -1,3 +1,10 @@
-<div class="media-hero">
-    MEDIA HERO
+<div class="hero" style="background-image: url('/assets/images/hero-bg.jpg');">
```

Comment:
You can use `{% static 'images/hero-bg.jpg' %}` (or equivalent) when linking static resources.

Although in this case we will more likely use an uploaded image using the `url` property on the image field in the model.